mava/networks.py

```diff
@@ -108,7 +111,41 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> distr
             jnp.finfo(jnp.float32).min,
         )
 
-        return distrax.Categorical(logits=masked_logits)
+        #  We transform this distribution with the `Identity()` transformation to
+        # keep the API identical to the ContinuousActionHead.
+        return IdentityTransformation(distribution=tfd.Categorical(logits=masked_logits))
+
+
+class ContinuousActionHead(nn.Module):
+    """ContinuousActionHead using a transformed Normal distribution.
+    Note: This network only handles the case where actions lie in the interval [-1, 1]."""
```

Comment:
```suggestion
    """ContinuousActionHead using a transformed Normal distribution.
    
    Note: This network only handles the case where actions lie in the interval [-1, 1].
    """
```

Reply:
Sorry I think was changed with the revert 😅